main.ts

```diff
@@ -37,6 +37,7 @@ export default class CustomJS extends Plugin {
     };
     this.app.workspace.onLayoutReady(async () => {
       await this.loadClasses();
+      window.customJS.app = this.app;

```

Comment:
I think it makes more sense to add this down in `loadClasses` where we initialize the customJS object.